docs/source/configuration/persisted-queries.mdx

```diff
@@ -6,8 +6,6 @@ minVersion: 1.25.0
 
 <GraphOSEnterpriseRequired />
 
-<PreviewFeature discordLink="https://discordapp.com/channels/1022972389463687228/1143901714173407342"/>
```

Comment:
Note that there are other references to "preview" on this page, but they are loaded via components in the `docs` repo.